torch/_export/verifier.py

```diff
@@ -133,6 +133,7 @@ def allowed_builtin_ops(self) -> List:
             operator.abs,
             math.ceil,
             math.floor,
+            math.trunc,
```

Comment:
This is used when the bilinear interpolate is decomposed (in the export test case)